micrometer-core/src/main/java/io/micrometer/core/instrument/Timer.java

```diff
@@ -143,7 +143,7 @@ static Builder builder(String name) {
         private String description;
         private final HistogramConfig.Builder histogramConfigBuilder;
 
-        private Builder(String name) {
+        protected Builder(String name) {
```

Comment:
@jkschneider This single line change is the only really 'required' change in Micrometer that will allow creating the other Tracing timer classes in other repos.

I would love to leave the Opentracing/openCensus implementations in Micrometer as Optional deps. I did research and the breaking API change that Adrian brought up will be properly shimmed, and should actually behave as a deprecation for a good while.